### ISSUE TITLE
Convert vendor sites to the new design

### ISSUE THREAD

**Issue by freejoe76** [denverpost/vendor-nav#4]
[X] Calendar
[X] Bill Tracker
[X] Election results
[ ] Movie times
[X] Crime map
[ ] Preps
[X] ScribbleLive

**Comment by freejoe76** [comment]
cc @schneidan

**Comment by schneidan** [comment]
@freejoe76 DFM just released their way of doing vendor navs, so I'll have to check that out for these remaining ones. Don't think it works for the sites where you have to upload the template to the vendor, though.

**Comment by freejoe76** [comment]
Huh, they had said that they weren't going to do that. That's great news!

**Comment by schneidan** [comment]
Yeah, it came totally out-of-the-blue. I haven't had time to investigate it in any depth yet, but it's on the road map!
